src/components/sdk/VideoTileGrid/index.tsx

```diff
@@ -38,11 +38,14 @@ interface Props extends BaseProps {
   noRemoteVideoView?: React.ReactNode;
   /** The layout of the grid. */
   layout?: Layout;
+  /** add local user name in localtile */
+  localuserName?:string;
 }
 
 export const VideoTileGrid: React.FC<Props> = ({
   noRemoteVideoView,
   layout = 'featured',
+  localuserName='You',
```

Comment:
Could you please keep the default "Me" string - if you'd like to change the username, feel free to provide the prop 'You'

Reply:
Also, I think that would have been a breaking change for the existing builders.